cves/kernel/CVE-2018-10940.yml

```diff
@@ -90,8 +94,8 @@ fixes:
   note:
 - commit: 9de4ee40547fd315d4a0ed1dd15a2fa3559ad707
   note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    Manually Confirmed. This fix simply changes the type-casting of the argument "arg" when checked against the capacity of the CD.
```

Comment:
I don't think this explanation is needed here.

Reply:
Why not? If it were longer and dragged-out I might agree, but I don't see the harm with a brief description of the fix.